conda_build/license_family.py

```diff
@@ -14,11 +13,11 @@
 GPL
 BSD
 MIT
-APACHE
+Apache
 PSF
-PUBLICDOMAIN
-PROPRIETARY
-OTHER
+Public-Domain
```

Comment:
Wouldn't this break anyone that was using `PUBLICDOMAIN`? Also do the case changes result in any problems?